### PR TITLE
completed wishlist and cart

### PR DESCRIPTION
**Pull request by archit7448**

### CODE REVIEW

**Review by purohitdheeraj — APPROVED** [review]
Looking good to merge👍

**Review by humzahasan — APPROVED** [review]
Good Strucutre. 
Appealing UI

Recommendation to place the count widget in center or maybe have button stacked upon each other. So the whitspace will be utalized. 

image

File: styles.css [hunk 1]
```diff
@@ -150,4 +150,51 @@ label:hover{
...
+}
+.card-cart{
+  width: fit-content;
+  padding: 1rem;
+  display: flex;
+}
+.card-cart img{
+  width: 20rem;
+  margin: 1rem;
+}
```
Comment (bhushannaikaroskar) [inline styles.css:171]: Use proper spacing between classes.

**Review by bhushannaikaroskar — APPROVED** [review]
Everything is perfectly fine I would suggest to use spacing wherever required fro a well structured code. Do proper spacing rest all I think its good to merge.

**Review by Muzammil-khan710 — APPROVED** [review]
After completing all the suggestions given above.. it's Good to merge

**Pull request merged**